core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -164,4 +164,83 @@ describe('Admin backend api service', () => {
 
     flushMicrotasks();
   }));
+
+  it('should Send SVGs to the backend.', fakeAsync(() => {
+    var successResponse = {
+      result: 'successfully updated'
+    };
+    var latexToSvgMapping = {
+      exp_id1: {
+        latex_string1: {
+          file: new Blob(),
+          dimensions: {
+            height: '4d456',
```

Comment:
This is a valid height...?

Reply:
refer to this [link](https://github.com/oppia/oppia/pull/9985#discussion_r460495301)